### DISCUSSION TITLE
Welcome to wad2bin Discussions!

### DISCUSSION THREAD

**Discussion by DarkMatterCore** [DarkMatterCore/wad2bin#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CosmicJenny** [comment]
I haven't found help elsewhere so I'm asking here.

How do I actually "assemble" the program? I'm an absolute beginner so I have no idea what I'm doing, I have no idea how any of the programs work.

**Reply by DarkMatterCore** [reply]
> I haven't found help elsewhere so I'm asking here.
> 
> How do I actually "assemble" the program? I'm an absolute beginner so I have no idea what I'm doing, I have no idea how any of the programs work.

It's okay, asking here is perfectly reasonable. This is why I enabled the Discussions tab in the first place.

You don't really need to "assemble" the program. You can download a ready to use executable file for Windows from the [releases page](https://github.com/DarkMatterCore/wad2bin/releases/latest).

Bear in mind this is a command-line tool, which means you can't just double-click the program and expect it to display a window with a graphical interface. You can, however:

1. Download the `gui.py` file from the releases page and place it alongside the downloaded `wad2bin.exe` file.
2. [Download and install Python](https://www.python.org/downloads).
3. Double-click the `gui.py` file to open a graphical frontend for wad2bin.

Hope that helps.

**Comment by Matzan61** [comment]
I have a little problem.
So basically I added the Wad files for JD4 DLC and when I click run nothing pops up in the Wii sd card
is there a way to fix it?

**Reply by DarkMatterCore** [reply]
> I have a little problem. So basically I added the Wad files for JD4 DLC and when I click run nothing pops up in the Wii sd card is there a way to fix it?

There probably is, but I'd say you just need to keep playing with the parent TID and the null key options, as well as using a different launch (c)IOS while you're at it. Your mileage may vary.

**Comment by MFLOPPA** [comment]
did wad2bin works on vWii?I just don't want to get brick on vWii.

**Reply by DarkMatterCore** [reply]
It does.

**Reply by MFLOPPA** [reply]
thanks

**Reply by larsenv** [reply]
You need Patched IOS80 Installer for vWii in order to use it

**Reply by MFLOPPA** [reply]
i already patch IOS80 and install cIOS 249,250 and 251 while hacking vWii

**Comment by MAYAMELEANA** [comment]
Just a wee problem
I have everything you said was necessary from previous comments, but when I click "Run", it says that the binary wasn't found... do you know what I need?

**Reply by DarkMatterCore** [reply]
Hi there. Please make sure the wad2bin executable is simply named `wad2bin`. Rename the file if needed.

**Reply by DarkMatterCore** [reply]
If you're using Windows, make sure to preserve the `.exe` extension as well -- e.g. `wad2bin.exe`.

**Reply by MAYAMELEANA** [reply]
It's working now, tysm!! I didn't realize it was that easy lol

**Reply by DarkMatterCore** [reply]
Glad to help.

**Reply by atiannarose93-png** [reply]
Hey, I'm having this same issue, and I've tried changing the name to just wad2bin and wad2bin.exe, and it still isn't working-- I've tried moving it to the root of my SD card, and it pops up with 'error opening directory' any clue what I'm doing wrong?
